spec.html

```diff
@@ -33745,6 +34823,7 @@ <h1>%TypedArray%.prototype.set ( _typedArray_ [ , _offset_ ] )</h1>
             1. Let _srcLength_ be _typedArray_.[[ArrayLength]].
             1. Let _srcByteOffset_ be _typedArray_.[[ByteOffset]].
             1. If _srcLength_ + _targetOffset_ &gt; _targetLength_, throw a *RangeError* exception.
+            1. If one of _srcType_ and _targetType_ contains the substring `"Big"` and the other does not, throw a *TypeError* exception.
```

Comment:
this seems like a suuuuuper brittle way to handle this. can we come up with a more reliable approach?

Reply:
I'll think in something better.